lib/install/stylesheets/tailwind.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  // Purge unused TailwindCSS styles
+  purge: {
+    enabled: ["production"].includes(process.env.NODE_ENV),
+    content: [
+      './**/*.html.erb',
+      './app/helpers/**/*.rb',
+      './app/javascript/**/*.js',
+    ],
+  },
+  darkMode: false, // or 'media' or 'class'
```

Comment:
unrelated to this PR, but should this be updated to match https://github.com/rails/tailwindcss-rails/blob/main/tailwind.config.js#L4 ?